redocly.yaml

```diff
@@ -0,0 +1,5 @@
+apis: 
+  main:
+    root: ./../../../do-not-exist.yaml
```

Comment:
```suggestion
    root: ./../../../does-not-exist.yaml
```

Reply:
This file got here accidentally. Thx for pointing out